client/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Life Aid</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.100.2/css/materialize.min.css">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Kurale" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Montserrat:400,500,900|Oswald:400,600,700,900|Open+Sans:800" rel="stylesheet">
+
+</head>
+
+<body>
+  <div id="root"></div>
+    <script type="text/javascript" src="https://code.jquery.com/jquery-3.2.1.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.100.1/js/materialize.min.js"></script>
```

Comment:
Use the material [npm module](https://www.npmjs.com/package/@material-ui/core)

Reply:
Removed